docs/resources/integration.md

```diff
@@ -144,3 +144,82 @@ Import is supported using the following syntax:
 # For example:
 $ terraform import juju_integration.wordpress_db 4b6bd192-13bb-489d-b7a7-06f6efc2928d:percona-cluster:server:wordpress:db
 ```
+
+Alternatively, it's possible to use the `import` block in your plan with an offer URL:
+```tf
+import {
+  to = juju_integration.this
+  # UUID obtained through a data source
+  id = "${data.juju_model.development.uuid}:percona-cluster:server:wordpress:db"
+}
+```
+
+### Cross-controller integration import
+
+It's also possible to import an integration between models on different controllers.
+
+```tf
+provider "juju" {
+  # In addition to the default controller, configure the offering one
+  offering_controllers = {
+    "db-controller" = {
+       controller_addresses = "..."
+       username             = "..."
+       password             = "..."
+       ca_certificate       = "..."
+    }
+  }
+}
+
+resource "juju_integration" "this" {
+  model_uuid = juju_model.development.uuid
+  via        = "10.0.0.0/24,10.0.1.0/24"
+
+  application {
+    # Controller name matches entry in `offering_controllers`
```

Comment:
```suggestion
    # Controller name matches entry in `provider.offering_controllers`
```

How about this? (Not sure if the `.` is standard notation or not, but felt the need to give the name of the block where this showed up.)